.changeset/hungry-eggs-move.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Split Changeset file into this ones to be more precise in the changelog files:
- xchain-util -> New isTokenAssetFunction
- xchain-evm and xchain-wallet -> Add function `awaitTxConfirmed` which waits until the approval is confirmed
- xchain-thorchain-amm, xchain-mayachain-amm -> ERC20 check for isRouterApprovedToSpend
- xchain-aggregator -> Support auto-approval for ERC-20 tokens

Reply:
Perfect, thank you!